tests/confirm-rpm.sh

```diff
@@ -60,8 +60,10 @@ fi
 
 # This semver regex just ignores the pre-release section without validating it
 version=$(rpm -qp --queryformat '%{VERSION}\n' "$pkgDir/$rpmName")
-if [ -z "$(echo "$version" | grep '^[0-9]\+\.[0-9]\+\.[0-9]\+')" ]
+if [ -n "$(echo "$version" | grep '^[0-9]_[0-9]')" ] || [ -n "$(echo "$version" | grep '^[0-9]\+\.[0-9]\+\.[0-9]\+')" ]
```

Comment:
Not needed if we fix to semver as I suggested above; or, rather than two `grep` statements, you could us `[_\.]`